pkg/runtime/application_config_provider.go

```diff
@@ -86,6 +87,10 @@ var externalEventsConfig = config.MustRegisterSection(externalEvents, &interface
 	Type: common.Local,
 })
 
+var cloudEventsConfig = config.MustRegisterSection(cloudEvents, &interfaces.CloudEventsConfig{
+	Type: common.Local,
```

Comment:
can we add sensible defaults for `ReconnectAttempts` and `ReconnectDelaySeconds`?

Reply:
ah okay, staying consistent is fine too